test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Order.html

```diff
@@ -41,6 +41,9 @@
                 <option value="Credit">Credit</option>
                 <option value="Check" selected="selected">Check</option>
             </select>
+            <select>
+                <option value="Check">Check</option>
+            </select>
```

Comment:
Do we have no way outside the functional tests to confirm the second `<select>` gets a different `Items` collection than the first? Hard to argue two `<select>` lists with overlapping `<option>` values and inconsistent tag helper use is mainline enough to show here.